lib/engine.js

```diff
@@ -18,6 +18,16 @@ const Mailer = require('inxt-service-mailer');
 const log = require('./logger');
 const ComplexClient = require('storj-complex').createClient;
 const MongoDBStorageAdapter = require('storj-mongodb-adapter');
+const NetworkMessageQueue = require('./server/queues/network_queue');
+
+const {
+  QUEUE_HOST: HOST,
+  QUEUE_PORT: PORT,
+  QUEUE_USERNAME: USER,
+  QUEUE_PASSWORD: PASS,
+} = process.env;
+
```

Comment:
For the moment, the env variables for the queue are being retrieved here from process.env

But unsure if there is a better way, or an ideal place we could retrieve them and just import them here.